content/docs/user-guide/experiment-management/checkpoints.md

```diff
@@ -2,24 +2,35 @@
 
 _New in DVC 2.0_
 
-To track successive steps in a longer experiment, you can register checkpoints
-from your code at runtime. This is especially helpful in machine learning, for
-example to track the progress in deep learning techniques such as evolving
-neural networks.
-
-_Checkpoint experiments_ track a series of variations (the checkpoints) and
-their execution can be stopped and resumed as needed. You interact with them
-using the `--rev` and `--reset` options of `dvc exp run` (see also the
-`checkpoint` field in `dvc.yaml` `outs`). They can help you
+To track successive steps in a longer machine learning experiment, you can
+register checkpoints from your code at runtime, for example to track the
+progress with deep learning techniques. They can help you
```

Comment:
I also started simplifying this Checkpoints guide in this PR. It wasn't the original intention but it helped integrate some details extracted from the `exp run` ref. in a way that makes more sense... Rel. #2769

Reply:
UPDATE: Moved Checkpoint guide and related changes to #3182.